src/types/plugin-input.ts

```diff
@@ -1,24 +1,12 @@
-import { EmitterWebhookEvent as WebhookEvent, EmitterWebhookEventName as WebhookEventName } from "@octokit/webhooks";
-import { StandardValidator } from "typebox-validators";
-import { SupportedEvents } from "./context";
 import { StaticDecode, Type as T } from "@sinclair/typebox";
-
-export interface PluginInputs<T extends WebhookEventName = SupportedEvents> {
-  stateId: string;
-  eventName: T;
-  eventPayload: WebhookEvent<T>["payload"];
-  settings: CommandQuerySettings;
-  authToken: string;
-  ref: string;
-}
+import { LOG_LEVEL } from "@ubiquity-os/ubiquity-os-logger";
 
 export const pluginSettingsSchema = T.Object({
   /**
    * Allows any user to query anyone else. If false, only org members can query others.
    */
-  allowPublicQuery: T.Boolean({ default: true }),
+  allowPublicQuery: T.Optional(T.Boolean({ default: true })),
+  logLevel: T.Optional(T.Enum(LOG_LEVEL, { default: LOG_LEVEL.INFO })),
```

Comment:
log level doesn't seem like a typical config option. This should be handled in the secrets/env of the plugin instead.

Reply:
I think every plugin should have this option because this allows choosing the level of logs you want during the run. I guess we could have it in the env, but wouldn't it be less user friendly? If you wanna be debugging something here, you could just add that in the configuration.

Reply:
I think it should be in the env because it's something that the plugin developer should set, not the plugin user (partner).

Reply:
@whilefoo https://github.com/ubiquity-os-marketplace/command-query/pull/17